spk/irssi/Makefile

```diff
@@ -9,7 +9,7 @@ MAINTAINER = SynoCommunity
 DESCRIPTION = Irssi is a terminal based IRC client for UNIX systems. It also supports SILC and ICB protocols via plugins.
 DISPLAY_NAME = Irssi
 STARTABLE = no
-CHANGELOG = "1. Update irssi to v1.2.3 released on 2021-04-11.<br/>2. Update openssl to 1.1.1k.<br/>3. Update glib to 2.66."
+CHANGELOG = "1. Update Irssi to v1.4.5.<br/>2. Bundle autodl-irssi plugin."
```

Comment:
I started to add a space before the <br/> in changelogs, because the changelog is showed as a one liner in DSM Package Center.
see

<img width="975" height="202" alt="Image" src="https://github.com/user-attachments/assets/ccc3bf32-4fd0-4b82-8201-13e4458f8e67" />

The current chanelog will be shown as
`1. Update Irssi to v1.4.5.2. Bundle autodl-irssi plugin.`

Until we find the real solution for multiline chandlogs we should insert at least one space before each line break.

Reply:
In DSM 6.2.4 this is not an issue

<img width="844" height="195" alt="Image" src="https://github.com/user-attachments/assets/2526e357-9fb8-4ca7-a660-ad8c2f26879b" />